hydrant/logserverhandler.py

```diff
@@ -7,8 +7,8 @@
 class LogServerHandler(logging.Handler):
     """Specialized logging handler capable of nesting json and passing auth"""
 
-    def __init__(self, url, jwt, level):
-        super().__init__(level)
+    def __init__(self, url, jwt):
+        super().__init__()
```

Comment:
It looks like you can pass `level` to [`logging.Handler()`](https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L867)?

```
>>> import logging
>>> help(logging.Handler)

Help on class Handler in module logging:

class Handler(Filterer)
 |  Handler(level=0)
 |  
 |  Handler instances dispatch logging events to specific destinations.
 |  
 |  The base handler class. Acts as a placeholder which defines the Handler
 |  interface. Handlers can optionally use Formatter instances to format
 |  records as desired. By default, no formatter is specified; in this case,
 |  the 'raw' message as determined by record.message is logged.
 |  
 |  Method resolution order:
 |      Handler
 |      Filterer
 |      builtins.object
 |  
 |  Methods defined here:
 |  
 |  __init__(self, level=0)
 |      Initializes the instance - basically setting the formatter to None
 |      and the filter list to empty.
```

Reply:
yes, you can pass it, as we were doing, but if the logger has a higher level set, it just silently ignores that detail, so I'm choosing to eliminate the confusion on our code.